lib/operations/options_operation.js

```diff
@@ -0,0 +1,39 @@
+'use strict';
+
+const OperationBase = require('./operation').OperationBase;
+const defineAspects = require('./common_functions').defineAspects;
+const handleCallback = require('../utils').handleCallback;
+const MongoError = require('mongodb-core').MongoError;
+
+class OptionsOperation extends OperationBase {
+  constructor(collection, options) {
+    super(options);
+
+    this.collection = collection;
+  }
+
+  static get aspects() {
```

Comment:
no need to define an `aspects` getter if you are using `defineAspects` (this happens in a few places)